instruqt/protecting-your-assets/03-proxying-private-images/assignment.md

```diff
@@ -0,0 +1,251 @@
+---
+slug: proxying-private-images
+id: 9o7xbvuzu2jq
+type: challenge
+title: Proxying Private Images
+teaser: Protect your private container images with the Replicated proxy registry
+notes:
+- type: text
+  contents: |
+    Share your private images without exposting your private registry to your customers
+tabs:
+- title: Vendor Portal
+  type: website
+  url: https://vendor.replicated.com
+  new_window: true
+- title: Slackernews Chart
+  type: code
+  hostname: shell
+  path: /home/replicant
+- title: Shell
+  type: terminal
+  hostname: shell
+  workdir: /home/replicant
+difficulty: basic
+timelimit: 600
+---
+
+One of the core features of the Replicated Platform is it's proxy registry. The
+proxy registry controls access to your images using the Replicated license
+This relieves you of the burden of managing authentication and authorization
+for the private images your application depends on. You provide Replicated with
+access and we manage the rest.
+
+Configuring a Private Registry
+==============================
+
+The first step in using the Replicated proxy registry is to provide access to
+your private registry for the Replicated Vendor Portal. This has already been
+done in the lab environment, so we're just going to review how it was set.
+
+Log into the Vendor Portal with the following credentials:
+
```

Comment:
I was this many days old when my instruct expired. Maybe I'm just slow, but this seems like an aggressively low timeout is that something you can increase?

Reply:
Oh crap I have to re-create my customer when that happens too!

Reply:
I've definitely increased the timeout. Chatting with @xavpaice about whether it's enough.

Reply:
Took me an hour to go over it in some depth, and if I was totally new to everything it might be double that.